src/views/blocks/rsvp.php

```diff
@@ -13,27 +13,52 @@
  *
  */
 
-$event_id = $this->get( 'post_id' );
-$tickets  = $this->get( 'tickets' );
+$event_id    = $this->get( 'post_id' );
+$tickets     = $this->get( 'tickets' );
+$num_tickets = count( $tickets );
 
-?>
-
-<?php $this->template( 'blocks/attendees/order-links', array( 'type' => 'RSVP' ) ); ?>
+// We don't display anything if there is no RSVP
+if ( empty( $num_tickets ) ) {
+	return false;
+}
 
-<div class="tribe-block tribe-block__rsvp">
+// Get active RSVPs
+$active_tickets = array();
 
-	<?php foreach ( $tickets as $ticket ) : ?>
+foreach ( $tickets as $ticket ) {
```

Comment:
We need to move this logic into the definition of getting RSVP items. We should keep Views as simple as possible.